components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/ExceptionCodes.java

```diff
@@ -432,11 +432,13 @@
     ALIAS_CANNOT_BE_EMPTY(900855, "The alias cannot be empty", 400, "The alias cannot be empty"),
 
     // API import/export related codes
-    ERROR_READING_META_DATA(900900, "Error while reading meta information from the definition", 400,
+    ERROR_READING_META_DATA(900907, "Error while reading meta information from the definition", 400,
```

Comment:
What's the reason for changing this error code?

Reply:
The same error code was defined in another place as well in the same file.